docs/core/config/auth/databases/ldap.md

```diff
@@ -681,157 +669,150 @@ Password checking attributes:
 Example:
 
 ```
-pass_attrs = \
-    =password=%{ldap:userPassword}, \
-    =user=%{ldap:mailRoutingAddress}, \
-    =home=%{ldap:homeDirectory}, \
-    =uid=%{ldap:uidNumber}, \
-    =gid=%{ldap:gidNumber}
+  fields {
+    password = %{ldap:userPassword}
+    user = %{ldap:mailRoutingAddress}
+    home = %{ldap:homeDirectory}
+    uid = %{ldap:uidNumber}
+    gid = %{ldap:gidNumber}
+  }
 ```
 
 There are also other special fields which can be returned. See
 [[link,passdb_extra_fields]].
 
 If you wish to avoid two LDAP lookups (passdb + userdb), you can use
 [[link,auth_prefetch]] instead of userdb ldap in `dovecot.conf`. In that
-case you'll also have to include `user_attrs` in `pass_attrs` field
+case you'll also have to include `user_attrs` in `fields` field
 prefixed with `userdb_` string.
+FIXME: WHAT ABOUT THIS ?
 
-### `pass_filter`
+### `ldap_filter`
 
 - Default: `<empty>`
 - Values: [[link,settings_types_string]]
 - *LDAP Authentication Only*
 
 Filter for password lookups (passdb lookup).
 
-Example: `pass_filter = (&(objectClass=posixAccount)(uid=%u))`
+Example: `ldap_filter = (&(objectClass=posixAccount)(uid=%u))`
 
-### `sasl_authz_id`
+### `ldap_auth_sasl_authz_id`
 
 - Default: `<empty>`
 - Values: [[link,settings_types_string]]
 
-SASL authorization ID, ie. the `dnpass` is for this "master user", but the
-`dn` is still the logged in user. Normally you want to keep this empty.
-
-### `sasl_bind`
-
-- Default: `no`
-- Values: [[link,settings_types_boolean]]
-
-Set `yes` to use SASL binding instead of the simple binding.
-
-Note that this changes [`ldap_version`](#ldap-version) automatically to be
-`3` if it's lower.
+SASL authorization ID, ie. the `ldap_auth_dn_password` is for this "master user", but the
+`ldap_auth_dn` is still the logged in user. Normally you want to keep this empty.
 
-### `sasl_mech`
+### `ldap_auth_sasl_mechanism`
 
 - Default: `<empty>`
 - Values: [[link,settings_types_string]]
 
 SASL mechanism names (a space-separated list of candidate mechanisms) to use.
 
-### `sasl_realm`
+### `ldap_auth_sasl_realm`
 
 - Default: `<empty>`
 - Values: [[link,settings_types_string]]
 
 SASL realm to use.
 
-### `scope`
+### `ldap_scope`
 
 - Default: `subtree`
 - Values:  `base`, `onelevel`, `subtree`
 
 This specifies the search scope.
 
-### `tls`
+### `ssl`
 
 - Default: `no`
 - Values: [[link,settings_types_boolean]]
 
 Set to `yes` to use TLS to connect to the LDAP server.
 
-### `tls_ca_cert_dir`
+### `ssl_ca_cert_dir`
```

Comment:
Remove these ssl_* settings, they are the same as the global ones. Possibly list the ones actually supported for LDAP.